backend/kale/common/snapshotutils.py

```diff
@@ -117,3 +117,22 @@ def get_snapshotclass_name(pvc_name, label_selector=""):
         if i["driver"] == provisioner:
             snapshotclass_name = i["metadata"]["name"]
     return snapshotclass_name
+
+
+def snapshot_pod():
+    """Take snapshots of the current Pod's PVCs."""
+    volumes = [(path, volume.name, size)
+               for path, volume, size in podutils.list_volumes()]
+    namespace = podutils.get_namespace()
+    pod_name = podutils.get_pod_name()
+    log.info("Taking a snapshot of pod %s in namespace %s ..."
+             % (pod_name, namespace))
+    snapshot_names = []
+    for i in volumes:
```

Comment:
Can you change `i` to something more idiomatic, e.g. `vol`? And maybe assign `vol[1]` to `name` so that it's clear what you are doing

Reply:
Yeah, when I was splitting up the PRs I noticed this as well. Will change it ASAP.